docs/docs/dev_docs/contracts/deploying.md

```diff
@@ -9,14 +9,46 @@ Once you have [compiled](./compiling.md) your contracts you can proceed to deplo
 
 ## Deploy
 
-To deploy the contracts we use the `deploy` command from `aztec-cli`:
+Contracts can be deployed using the `aztec-cli` or using the `aztec.js` library. 
+
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Tabs groupId="deployment-methods">
+<TabItem value="cli" label="Aztec CLI">
 
 ```bash
 aztec-cli deploy /path/to/contract/abi.json
 ```
 
-### Arguments
-This command takes 1 mandatory argument which is the path to the contract ABI file in a JSON format (e.g. `contracts/target/PrivateToken.json`). Alternatively you can pass the name of an example contract as exported by `@aztec/noir-contracts` (run `aztec-cli example-contracts` to see the full list of contracts available).
+</TabItem>
+<TabItem value="js" label="Aztec.js">
+
+Pre-requisite - Generate type-safe typescript classes for your contract when compiling:
```

Comment:
Is there a way to do this from aztec js as well? Ideally, want to support doing it fully programatically with aztec js and not having to touch the cli for some tests etc.

Reply:
https://github.com/AztecProtocol/aztec-packages/blob/master/docs/docs/dev_docs/contracts/compiling.md#compile-using-nodejs

@spalladino has thought of everything :)

Reply:
nope there isn't afaik :(